pkg/constants/constants.go

```diff
@@ -220,6 +221,31 @@ var (
 	}
 )
 
+var (
+	//RawIngressNamePrefix ...
+	RawIngressNamePrefix = KFServingName + "-ingress"
+)
+
+// raw k8s deployment, resource exist check result
+type CheckResultType int
+
+const (
+	CheckResultCreate  CheckResultType = 0
+	CheckResultUpdate  CheckResultType = 1
+	CheckResultExisted CheckResultType = 2
+	CheckResultUnknown CheckResultType = 3
+)
+
+// GetRawServiceLabel generate native service label
+func GetRawServiceLabel(service string, namespace string) string {
+	return "isvc." + namespace + "." + service
```

Comment:
I think we do not need the prefix and namespace, the services are namespace scoped.

Reply:
right, this is useless